responses/fr/HassGetWeather.yaml

```diff
@@ -3,9 +3,24 @@ responses:
   intents:
     HassGetWeather:
       default: >
-        {# Reuse weather condition already formatted and translated #}
-        {# just invert telling the condition then temperature #}
-        {{ state.state }}
+        {% set weather_condition = {
+          'clear-night': 'Nuit dégagée',
+          'cloudy': 'Nuageux',
+          'exceptional': 'Conditions exceptionnelles',
+          'fog': 'Brouillard',
+          'hail': 'Grêle',
+          'lightning': 'Orageux',
+          'lightning-rainy': 'Orageux avec de la pluie',
+          'partlycloudy': 'Éclaircies',
+          'pouring': 'Averses',
+          'rainy': 'Pluvieux',
+          'snowy': 'Neigeux',
+          'snowy-rainy': 'Neigeux avec pluie mêlée',
+          'sunny': 'Ensoleillé',
+          'windy': 'Venteux',
+          'windy-variant': 'Venteux et nuageux',
+        } %}
+        {{ weather_condition.get(state.state | string, "") }}
```

Comment:
I assumed the future behaviour already, so I don't "lower" like most weather intents do today.